File: classify-server/src/main/java/io/ermdev/alice/entity/Schedule.java
package io.ermdev.alice.entity;

import javax.persistence.*;
import java.sql.Time;

@Table(name = "tblschedule")
@Entity
public class Schedule {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String day;
    private Time startTime;
    private Time endTime;
    private Double hours;
    private String room;

    @JoinColumn(name = "classId")
    @ManyToOne
    private Class _class ;

    public Schedule(){}


}

File: src/main/kotlin/io/ermdev/classify/data/entity/Schedule.kt
package io.ermdev.classify.data.entity

import java.sql.Time
import javax.persistence.*

@Entity
@Table(name = "tbl_schedule")
class Schedule(@Id
               @GeneratedValue(strategy = GenerationType.AUTO)
               var id: Long = 0,

               var day: String = "",

               var room: String = "",

               var start: Time = Time(System.currentTimeMillis()),

               var end: Time = Time(System.currentTimeMillis()),

               @ManyToOne(cascade = [CascadeType.PERSIST])
               @JoinColumn(name = "lessonId", nullable = false)
               var lesson: Lesson = Lesson()) {

    override fun toString(): String {
        return "start = $start,  end = $end"
    }
}